src/dump-syms.ts

```diff
@@ -1,18 +1,28 @@
 import util from 'util';
 import path from 'path';
+import fs from 'fs/promises'
 import * as child_process from 'child_process';
 const exec = util.promisify(child_process.exec);
 
 export async function dumpSyms(filePath: string): Promise<{ stdout: string, stderr: string}> {
     const { platform } = process;
-    const isLinux = platform === 'linux';
-    
-    if (!isLinux) {
+    const dumpSymsPath = path.join(__dirname, `../bin/${platform}/dump_syms`);
+    const supported = await exists(dumpSymsPath);
```

Comment:
[QUESTION] any reason you don't use I think....fs.existsSync ?

Reply:
I think existsSync has been deprecated?

Reply:
ah! my mistake, it seems only fs.exists has been deprecated:

```
fs.exists() is deprecated, but fs.existsSync() is not
```

from https://nodejs.org/api/fs.html#fsexistssyncpath